openapi.yml

```diff
@@ -3683,6 +3683,273 @@ paths:
           description: The Location does not exist
       operationId: getLocation
       description: Gets details of a single instance of `Location`.
+  /packages:
+    get:
+      summary: Get packages
+      tags:
+        - Packages
+      responses:
+        '200':
+          description: OK
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: ./models/PaginatedPackagesList.v1.yaml
+              examples:
+                example:
+                  value:
+                    packages:
+                      - id: 70
+                        package_state: picked_up
+                        recipient:
+                          id: 5
+                          email: tsala@tractionguest.com
```

Comment:
not sure we should use real emails in the public API unless we're wanting people to contact them?